beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -224,6 +218,8 @@ pub struct BeaconChain<T: BeaconChainTypes> {
     pub disabled_forks: Vec<String>,
     /// Logging to CLI, etc.
     pub(crate) log: Logger,
+    /// Arbitrary bytes included in the blocks.
```

Comment:
Wrote arbitrary here (because eth2 spec says so), but atm it's only &str. Should I be more precise?

Reply:
I think it's fine as-is, at this point `Graffiti` is just arbitrary bytes :)